demos/http3/ossl-nghttp3-demo-server.c

```diff
@@ -158,9 +157,13 @@ static int read_from_ssl_ids(nghttp3_conn *h3conn, struct h3ssl *h3ssl) {
     if (stream != NULL) {
       printf("=> Received connection on %lld\n", (unsigned long long) SSL_get_stream_id(stream));
       add_id(SSL_get_stream_id(stream), stream, h3ssl);
-      return 1; /* loop until we have all the streams */
+      printf("read_from_ssl_ids %ld events\n", (unsigned long) result_count);
+      if (result_count == 1) {
+        return 1; /* loop until we have all the streams */
+      }
+    } else {
+      return -1; /* something is wrong */
```

Comment:
Wouldn't it be better to invert the `if` condition and get rid of the `else` branch. Meaning

```.c
if (stream == NULL) {
    return -1; /* something is wrong */
}

printf("=> Received connection on %lld\n", (unsigned long long) SSL_get_stream_id(stream));
add_id(SSL_get_stream_id(stream), stream, h3ssl);
printf("read_from_ssl_ids %ld events\n", (unsigned long) result_count);
if (result_count == 1) {
    return 1; /* loop until we have all the streams */
}
```